### DISCUSSION TITLE
Unable to use GitHub to Clone the Macports repo

### DISCUSSION THREAD

**Discussion by RobK88** [community/community#120595] [Repositories]
### Select Topic Area

Bug

### Body

I am unable to create a new repo by cloning https://github.com/macports/macports-ports.git into my personal Github webpage.

I get an error.

The repo is there but I cannot open it or delete it.

**Comment by RobK88** [comment]
Finally, I was able to open and delete the repo.  Do not know why I got the error in the first place.

You can close this issue off.

**Reply by ettaboyle** [reply]
Done! Thanks for following up.
